### PR TITLE
Kudriavtsev_Yaroslav_Task1

### PR DESCRIPTION
**Pull request by IReeZoxI**

### CODE REVIEW

**Review by Dmitriy-NK — COMMENTED** [review]
File structure is OK. No additional change is required.

**Review by Dmitriy-NK — COMMENTED** [review]
1. Зачем дублировать часть с 5 графиками?
2. В разделе "graph down by gender" отсутствует график нормированного(в долях единицы) распределения выживших в двух подмножествах 'female','male'. И просьба добавить несколько предложений по своей оценке после "Add your summary about recieved data for raw value counts and proportionally" в том же разделе
3. В следующем разделе задания (Pclass structure) была просьба построить данные для 2 подмножеств (объединенный 1 и 2 класс) и 3 класс. Также добавьте график с учетом значения Выживших и нет в данных подмножествах.
4. Внимательней проанализируйте данные тепловой карты для confusion matrix, они расходятся с датасетом. по умолчанию sklearn.metrics.confusion_matrix считает метку 0 - положительной (это связано с возможностью классификации с многими метками) так собственный порядок можно задать в аргументе labels, в нашем случае [1,0]  [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
5.  Для построения ROC Кривых использованы некорректные данные. Данный момент поднимался во время практикума в задании с ирисами. Для построения соответствующего графика нужны данные из predict_proba метода
